addons/l10n_ec/data/account_tax_group_data.xml

```diff
@@ -51,19 +58,33 @@
             <field name="country_id" ref="base.ec"/>
         </record>
 
-        <record id="tax_group_withhold_vat" model="account.tax.group">
-            <field name="name">VAT Withhold</field>
-            <field name="l10n_ec_type">withhold_vat</field>
+        <record id="tax_group_withhold_vat_sale" model="account.tax.group">
+            <field name="name">VAT Withhold on Sales</field>
+            <field name="l10n_ec_type">withhold_vat_sale</field>
+            <field name="sequence">80</field>
+            <field name="country_id" ref="base.ec"/>
+        </record>
+        
+        <record id="tax_group_withhold_vat_purchase" model="account.tax.group">
+            <field name="name">VAT Withhold on Purchases</field>
```

Comment:
@pepetreshere Why do we only have one tax group per type?  What if we have different percentages on the same withhold?

Reply:
@jco-odoo there is no problem, the vat withhold is a fixed percentage (for instance 30%) of the vat amount..... no matter where the vat amount comes from. In other words, the computation of the withhold departs from the invoice subtotals where the total vat is already computed, no need to differentiate among the different percentages of vat. This is easily achieved through our decoupled design for withholds (which are separated from the invoice)

Reply:
@pepetreshere Having thought about it with the views yesterday, I think you are right, but we should remove the on Purchases part in the name because it is already obvious from the journal e.g.
```suggestion
            <field name="name">VAT Withhold</field>
```

Reply:
@jco-odoo I recommend not changing the name of the tax group because:
- Then there will be 2 groups with the same name (VAT Withhold on Sales and VAT Withhold on Purchases)
- The user will have no way to differentiate between those 2 groups when manually creating new taxes
- It provides no advantage to the end user, it is true that the journal might be used to differentiate but there is no need for that differentiation in the acount.move form nor on the wizard form, the text of the tax group is never shown in those views.

Please confirm if you insist in changing the name (I guess for both tax groups, and maybe even for both of the profit withhold groups)

Reply:
Could we just put (p) instead of purchass?  It takes a lot of space. (if you check in the withholding view)

Reply:
@jco-odoo Let's review it together, this is shown solely on the tax group menu inside configuration (this menu is even hidden for non-developer users). It is not shown in any other menu, form, tree, subtotal, or anywhere that I can recall (please note that the withhold subtotals are computed as a JSON field and the label is computed and not based on group name)

I would suggest that we keep the as it is, the full name that is most descriptive and will limit configuration errors.

Reply:
@jco-odoo please confirm if we could leave it as it is as it is not appearing anywhere and having full text will avoid configuration errors when new taxes are created